packages/vscode-extension/src/debug/taskTerminal/prerequisiteTaskTerminal.ts

```diff
@@ -77,6 +87,7 @@ export class PrerequisiteTaskTerminal extends BaseTaskTerminal {
     const res = await checkAndInstallForTask(
       this.args.prerequisites ?? [],
       this.args.portOccupancy,
+      this.args.installOptions,
```

Comment:
Can we just use `vxTestApp` as the arg name? IMO, it's clear that each prerequisite just use its name as option name:
``` json
{
    "prerequisites": [
        "AAA",
        "BBB",
        "..."
    ],
    "AAA": {},
    "BBB": {},
    "..."
}
```

Reply:
The implementation can convert them to different types of options, but for UX less level is clearer.

Reply:
Updated